<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HslDialog</class>
 <widget class="QDialog" name="HslDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>色相/饱和度/亮度调整</string>
  </property>
  <widget class="MSlider" name="hSlider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-180</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="hlabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>色相：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string>#000</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="slabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>饱和度：</string>
   </property>
  </widget>
  <widget class="MSlider" name="sSlider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="sEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string>#000</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="llabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>亮度：</string>
   </property>
  </widget>
  <widget class="MSlider" name="lSlider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string>#000</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>95</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="confirmButton">
      <property name="text">
       <string>确定</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MSlider</class>
   <extends>QSlider</extends>
   <header>mslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
